pos_pricelist/models/account_fiscal_position.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class AccountFiscalPositionTax(models.Model):
+    _inherit = "account.fiscal.position.tax"
+
+    company_id = fields.Many2one(
+        related="position_id.company_id", string="Company")
```

Comment:
Thanks for your clarification. 

> Security rules are applied without problem anyway.

It's the most important ! I didn't know that security rules could be applied on non stored computed field. Thanks.